pocs/camera/camera.py

```diff
@@ -80,7 +102,7 @@ def process_exposure(self, *args, **kwargs):
         raise NotImplementedError
 
     def __str__(self):
-        return "{}({}) on {}".format(self.name, self.uid, self.port)
+        return "{} ({}) on {}".format(self.name, self.uid, self.port)
```

Comment:
It does, though the resulting string representation could get pretty long. The `Focuser` classes have their own `name`, `uid` and `port` attributes which are included in their string representations.  Perhaps it should be just `"{} ({}) on {} with {} focuser".format(self.name, self.uid, self.port, self.focuser.name)` for a `Camera` with a `Focuser`?